### DISCUSSION TITLE
Add -Og optimization flag

### DISCUSSION THREAD

**Discussion by k1nder10** [emscripten-core/emscripten#18123] [Ideas]
According to https://emscripten.org/docs/optimizing/Optimizing-Code.html emscripten doesn't have -Og flag though it'd be pretty useful to have one. We're porting a big project to wasm and have to balance between the wasm_apply_data_relocs size and keeping debug symbols. It means that we want to optimize the wasm code (we have to do so to not exceed the function size limit) and have the code debuggable.

**Comment by sbc100** [comment]
I think the lack of `-Og` support was simply and oversight.  We should add it.
